### PR TITLE
Fix mypy failure by ingoring line

### PR DESCRIPTION
**Pull request by svenvanderburg**

### CODE REVIEW

File: proof_of_concept/policy_replication.py [hunk 1]
```diff
@@ -84,7 +84,7 @@ def _update(self) -> None:
         # removed ones that disappeared
         removed_servers = [
                 server for server in self._other_stores
-                if server not in list(zip(*new_servers))[1]]
+                if server not in list(zip(*new_servers))[1]]  # type: ignore
```
Comment (svenvanderburg) [inline proof_of_concept/policy_replication.py:87]: Mypy is complaining here with: "error: Unsupported right operand type for in ("object")". From a small inspection this code does not seem wrong.

Comment (svenvanderburg) [reply proof_of_concept/policy_replication.py:87]: i.e. the result of `list(zip(*new_servers))[1]` is a tuple

Comment (LourensVeen) [reply proof_of_concept/policy_replication.py:87]: Agreed, this looks like a mypy issue, it has these things from time to time.

**Review by LourensVeen — APPROVED** [review]
Looks fine, lgtm.

**Pull request merged**